sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -277,14 +293,21 @@ export class ReceiverImpl<ReceivedMessageT extends ReceivedMessage | ReceivedMes
     }
 
     const receiveMessages = async () => {
-      if (!this._context.batchingReceiver || !this._context.batchingReceiver.isOpen()) {
+      if (
+        !this._batchingReceiver ||
+        !this._context.batchingReceivers[this._batchingReceiver.name]
```

Comment:
Previous check used to check if the batchingReceiver was also open which seems like a good check. Any reason to remove it?

Reply:
Internally, if it`isOpen()` returns false, we initialize the link anyway right? I think that is why I removed the check. I can bring it back to maintain status quo and log an issue if needed

Reply:
Honestly, this one isn't the clearest of sequences. From my read it looks like context.batchingReceiver will just get unconditionally assigned over (potentially orphaning an open connection).

Reply:
Went over this offline with @richardpark-msft 
His concern that we will end up creating a new batching receiver even if the existing one is open is taken care of after we went over the if conditions above one more time in details :)